src/MandrillAdapter.php

```diff
@@ -42,6 +42,11 @@ public static function displayName(): string
      */
     public $apiKey;
 
+    /**
+     * @var string The subaccount that should be used
+     */
+    public $subAccount;
```

Comment:
If Mandrill calls them “Subaccounts” (one word) then this should be `$subaccount` (lowercase `a`).

Reply:
(and ditto for other places you have `subAccount`.)